src/apps/containers/components/EditMenu.tsx

```diff
@@ -1,15 +1,24 @@
-import {Fab, Menu, MenuItem, Tooltip} from "@material-ui/core";
+import {Button, Fab, Menu, MenuItem, Tooltip, makeStyles} from "@material-ui/core";
 import {MoreVert as MenuIcon} from "@material-ui/icons";
 import {Link} from "react-router-dom";
 import React from "react";
 import {useAnchor} from "../../../utils";
 
 interface Props {
     backUrl: string;
+    hasDelete?: boolean;
+    openModal?: () => void;
```

Comment:
I think this could be named better, e.g., `confirmDelete` or `handleDelete`. Something that makes it clear what it's tied to.